src/components/map/components/DeparturesDialogSheet.tsx

```diff
@@ -150,6 +193,24 @@ const useBottomSheetStyles = StyleSheet.createThemeHook((theme) => ({
   departuresContainer: {
     flex: 1,
   },
+  buttonsContainer: {
+    padding: theme.spacings.medium,
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'center',
```

Comment:
Are `alignItems` and `justifyContent` necessary here?

Reply:
In the design, the text appears to be centred.
<img width="755" alt="image" src="https://user-images.githubusercontent.com/43450882/199743360-6f8161d3-ac93-4dd8-9a48-ef2711479d0d.png">

Reply:
But this is the `buttonsContainer`?